payment_alipay/data/alipay.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="0">
+
+        <record id="payment_acquirer_alipay" model="payment.acquirer">
+            <field name="name">Alipay</field>
+            <field name="provider">alipay</field>
+            <field name="company_id" ref="base.main_company"/>
+            <field name="view_template_id" ref="alipay_acquirer_button"/> 
+            <field name="environment">test</field>
+            <field name="pre_msg"><![CDATA[
+<p>You will be redirected to the alipay website after clicking on the payment button.</p>]]></field>
+            <field name="alipay_pid">2088701568026380</field>
```

Comment:
Is it demo data?
If not this data should be a property in the website setup page or payment acquirer.
remove the specific data from the company